urls/readme.md

```diff
@@ -0,0 +1,76 @@
+# `Url` a utility class to parse & manupulate URLs
```

Comment:
Nice! I really like this method of working.

Even though there are loads of url libs out
there, by rolling your own here as a separate
project, you can really solve our specific needs,
understand the problem domain better, and then
if you come across an external lib that ends up
being good enough, dropping this sub project
and hooking up that lib is a breeze.

Reply:
Yeah definitely! I was wondering if it should go in `clientUtils` but I liked that `Patch` is its own thing and has no OWID-specific functionality.